lib/ui/compositing/scene_builder.cc

```diff
@@ -18,6 +18,7 @@
 #include "flutter/flow/layers/transform_layer.h"
 #include "flutter/lib/ui/painting/matrix.h"
 #include "flutter/lib/ui/painting/shader.h"
+#include "lib/ftl/build_config.h"
```

Comment:
I'd move ` #include "flutter/flow/layers/child_scene_layer.h"` into an ifdef block for OS_FUCHSIA because that header doesn't really exist on non-fuchsia.